tests/integration/records/test_views.py

```diff
@@ -669,10 +655,15 @@ def test_institutions_search_json_get(api_client, db, create_record_factory):
 
 
 def test_literature_facets_collaboration(api_client, db, create_record_factory):
-    data = {"collaborations": [{"value": "Alice"}, {"value": "Collab"}]}
-    record_1 = create_record_factory("lit", data=data, with_indexing=True)
-    data = {"collaborations": [{"value": "Alice"}]}
-    record_2 = create_record_factory("lit", data=data, with_indexing=True)
+    data_1 = {
```

Comment:
this should be a ``create_record`` as well + you can always skip ``document_type``, ``control_number``, ``_collections```

Reply:
ditto